src/gui/joycontrolstickeditdialog.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>702</width>
-    <height>570</height>
+    <height>568</height>
```

Comment:
Why this file is edited?

Reply:
Yes, this was accidentally committed. I removed it.